### PR TITLE
Make trainee selector for all attendance for AMs

### PR DESCRIPTION
**Pull request by billyeh**
Before: under attendance, AMs could view/input trainee rolls only. 
Now: they can view/input trainee rolls, leaveslips, and summary.

### CODE REVIEW

**Comment by billyeh** [comment]
This guy's ready for review @joeyu23 or @yipingbenjaminliu? Should be some light code reading if you need a break 😉  Whenever you have time is fine, it's not that high priority.

File: libraries/react/scripts/components/GroupSlipForm.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const GroupSlipForm = ({...props}) => {
         value={props.form}
         onChange={props.changeGroupSlipForm}
         onSubmit={props.postGroupSlip}
+        delay={100}
```
Comment (joeyu23) [inline libraries/react/scripts/components/GroupSlipForm.js]: Nice fix to the previous validation issue that came up with this feature change.

**Comment by joeyu23** [comment]
LGTM 😄  Joe Yu approves of joeyubranch

**Comment by billyeh** [comment]
Ok, thanks Joe! Let's merge it then.

**Pull request merged**